independent-publisher-connectors/ShipStationIP/apiDefinition.swagger.json

```diff
@@ -0,0 +1,398 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "ShipStation IP",
+    "description": "The API is a great way to get data directly to and from ShipStation, like creating orders, updating products, and querying order, shipment and customer data.",
+    "contact": {
+      "name": "ShipStation",
+      "url": "https://shipstation.com"

```

Comment:
Hello @SmolSoftBoi,
I hope you are doing well.
Can you please add an email address? We have a weekly connector telemetry report for our independent publishers. To start getting this report, please add your email in the swagger file and submit a quick update.
Please let me know if you have any questions.
Thank you so much for you hard work with us.
Best regards,
Amjed Ayoub

Reply:
Added in pull request #1813